src/redisgears_python.c

```diff
@@ -146,10 +172,19 @@ static bool PythonRequirementCtx_DownloadRequirement(PythonRequirementCtx* req){
     }
     closedir(dr);
 
-    return true;
+    req->isDownloaded = true;
+
+done:
+    pthread_mutex_unlock(&req->installationLock);
```

Comment:
should we also unlock before we assert (on the opendir)?

Reply:
Assert is not recoverable so why unlock?